### DISCUSSION TITLE
739

### DISCUSSION THREAD

**Discussion by giscus** [jojoldu/blog-comments#920] [Comments]
# 739

요즘 받아 보는 고민 중에 "가장 좋아하는 일이 프로그래밍이 아니다는 것을 인정하기 힘들다"는 것이 있다. 예를 들면 이런 것인데, 본인은 시간이 날때마다 그림을 그리는데 그 시간이 너무 재밌다는 것이다. 그래서 그림 그리러 가는 시간이 항상 기다려지고 빨리 퇴근하고 싶은 생각을 계속 한다는 것이다. 그래서 주변에서는 프로그래밍보다 그림 그리는 것을 더 좋아하는 것 같다고 농담으로 이야기하는데, 자긴 그걸 인정할 수 없다는 것이다. 근데 나는 아무리 고민을 들어봐도 프로그래밍 보다 그림을 그리는 것을 더 좋아한다고 느꼈다. 그리고 그게 왜 문제지? 라는 생각을 했다. 좀 더 고민을 들어보니 프로그래머로서 프로그래밍보다 더 좋아하는게 있어선 안된다고 믿고 있던 것이였다. 커리어 초반에 같은 팀에 있던 사수..

https://jojoldu.tistory.com/739

**Comment by J-A-Y2** [comment]
꾸준함을 유지할 수 있다면 가장 잘 맞는 직업이다. 너무나도 직관적인 글인것 같습니다~!

**Comment by murphybread** [comment]
최근 생각하던 주제라 순식간에 읽었네요 ㅎㅎ

특히나 "감정과 상관 없이 매일 매일 계속할 수 있다면"부분이 제일 공감이 됐는데 그래서 직업은 "좋아하는걸 직업을 가져라"라는 말 보단 "버틸 수 있는 것을 직업으로가져라"라는 말이 더 어울린다고 생각하게 되던 시기였거든요 ㅎㅎ

아침부터 좋은 글 읽고 화이팅합니다~

**Comment by ahzkwid** [comment]
역시 일과 취미는 다르죠. 즐겁다면 일이 아니라 놀이니까요. 일이 즐거워야 한다고 생각하는 사람은 일하다가 조금만 힘들어도 빈둥대려는 기색이 있기도 합니다.

**Comment by RedpingDev** [comment]
캬....저도 항상 하는 고민이였는데, 다른분의 생각과 관점을 알 수 있어서 정말 감사한 글이였습니다.
덕분에 또다른 관점에서 생각 할 수 있고 자신에게 너그러워 질 수 있게 되었습니다.

**Comment by gnoyes-mik** [comment]
"프로그래밍은 내가 좋아하는 일이라, 이 일이라면 꾸준히 할 수 있을 것 같다!"고 생각해오다가
"그치만 이보다 더 내가 좋아하고 몰입할 수 있는 일이 있지 않을까?" 라는 고민이 들던 요즘이었는데
마침 이 글을 보게되어 두 번, 세 번 읽었네요 ㅎㅎ

글의 커버 사진이 무라카미 하루키인 이유는 향로님이 가장 좋아하는 작가라서 그런걸까요? ㅎㅎ

**Comment by Chadolbaegi128** [comment]
좋아하는 것을 직업으로 삼으면 베스트이겠지만 꾸준히 할 수 있어야 한다는 게 중요한 거 같습니다.

**Comment by fortune-man** [comment]
잘 읽고 갑니당
